sci-geosciences/grass/grass-7.8.4.ebuild

```diff
@@ -0,0 +1,273 @@
+			ewarn "   eselect lapack set <impl>"
+			ewarn "where <impl> is atlas, threaded-atlas or reference"
+			die "setup failed"
+		fi
+	fi
+
+	if use blas; then
+		local myblas=$(eselect blas show)
+		if [[ -z "${myblas/.*reference.*/}" ]] && \
+			[[ -z "${myblas/.*atlas.*/}" ]]; then
+			ewarn "You need to set blas to atlas or reference. Do:"
+			ewarn "   eselect blas set <impl>"
+			ewarn "where <impl> is atlas, threaded-atlas or reference"
+			die "setup failed"
+		fi
+	fi
+
+	python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	# Fix unversioned python calls
+	sed -e "s:=python3:=${EPYTHON}:" -i "${S}/lib/init/grass.sh" || die
+	sed -e "s:= python3:= ${EPYTHON}:" -i "${S}/include/Make/Platform.make.in" || die
+
+	default
+	eautoreconf
+
+	ebegin "Fixing python shebangs"
+	python_fix_shebang -q "${S}"
+	eend $?
+
+	# For testsuite, see https://bugs.gentoo.org/show_bug.cgi?id=500580#c3
+	shopt -s nullglob
+	local mesa_cards=$(echo -n /dev/dri/card* /dev/dri/render* | sed 's/ /:/g')
+	if test -n "${mesa_cards}"; then
+		addpredict "${mesa_cards}"
+	fi
+	local ati_cards=$(echo -n /dev/ati/card* | sed 's/ /:/g')
+	if test -n "${ati_cards}"; then
+		addpredict "${ati_cards}"
+	fi
+	shopt -u nullglob
+	addpredict /dev/nvidiactl
+
+	eapply_user
+}
+
+src_configure() {
+	if use X; then
+		local WX_BUILD=yes
+		setup-wxwidgets
+	fi
+
+	addwrite /dev/dri/renderD128
+
+	local myeconfargs=(
+		--enable-shared
+		--disable-w11
+		--without-opendwg
+		--with-regex
+		--with-gdal="${EPREFIX}"/usr/bin/gdal-config
+		--with-proj-includes="${EPREFIX}"/usr/include/proj
+		--with-proj-libs="${EPREFIX}"/usr/$(get_libdir)
+		--with-proj-share="${EPREFIX}"/usr/share/proj/
+		$(use_with cxx)
+		$(use_with tiff)
+		$(use_with png)
+		$(use_with postgres)
+		$(use_with mysql)
+		$(use_with mysql mysql-includes "${EPREFIX}"/usr/include/mysql)
+		$(use_with sqlite)
+		$(use_with opengl)
+		$(use_with odbc)
+		$(use_with fftw)
+		$(use_with blas)
+		$(use_with lapack)
+		$(use_with X cairo)
+		$(use_with truetype freetype)
+		$(use_with truetype freetype-includes "${EPREFIX}"/usr/include/freetype2)
+		$(use_with nls)
+		$(use_with readline)
+		$(use_with threads pthread)
+		$(use_with openmp)
+		$(use_with opencl)
+		$(use_with liblas liblas "${EPREFIX}"/usr/bin/liblas-config)
+		$(use_with X wxwidgets "${WX_CONFIG}")
+		$(use_with netcdf netcdf "${EPREFIX}"/usr/bin/nc-config)
+		$(use_with geos geos "${EPREFIX}"/usr/bin/geos-config)
+		$(use_with X x)
+		$(use_with zstd)
+	)
+	econf "${myeconfargs[@]}"
+}
+
+src_compile() {
+	# we don't want to link against embedded mysql lib
+	emake CC="$(tc-getCC)" MYSQLDLIB=""
+}
+
+src_install() {
+	emake DESTDIR="${ED}" \
+		INST_DIR="${ED}"/usr/$(get_libdir)/${MY_PM} \
+		prefix="${ED}"/usr/ BINDIR="${ED}"/usr/bin \
+		PREFIX="${ED}"/usr/ \
+		install
+
+	pushd "${ED}"/usr/$(get_libdir)/${MY_PM} >/dev/null || die
+
+	local HTML_DOCS=( docs/html/. )
+	einstalldocs
+
+	# manuals
+	dodir /usr/share/man/man1
+	rm -rf man/ || die
```

Comment:
Drop the `-f` from here, we'd like to spot any errors in the ebuild.

Reply:
Thank you for the check. Changing `WX_GTK_VER` to `3.0-gtk3` fixed the build error. Lines 205-207 proven to be obsolete.